project_generator/tools/tool.py

```diff
@@ -15,8 +15,9 @@
 import os
 import logging
 
-from os.path import join, dirname
-from jinja2 import Template
+from os.path import join, dirname, abspath
```

Comment:
nitpick: this change should be a separate commit with a description of using jinja enviroment and file loader